src/App.js

```diff
@@ -70,16 +73,18 @@ class App extends Component {
     }
 
     return (
-      <React.Fragment>
-        <NotificationsPortal />
-        <Main style={ { marginLeft: 0, padding: 0 } }>
-          <Grid style={ { minHeight: MIN_SCREEN_HEIGHT } }>
-            <GridItem sm={ 12 }>
-              <Routes childProps={ this.props } />
-            </GridItem>
-          </Grid>
-        </Main>
-      </React.Fragment>
+      <IntlProvider locale="en">
+        <Fragment>
```

Comment:
Do you need the `Fragment` inside the `IntlProvider`?

Reply:
You can have only one child in the provider. (Same as Router used to be)

Reply:
I see. I just hit into that trying to do the same in Source.